sdk/eventhub/Azure.Messaging.EventHubs.Processor/README.md

```diff
@@ -88,7 +198,16 @@ Each of the samples is self-contained and focused on illustrating one specific s
 The available samples are:
 
 - [Hello world](./samples/Sample01_HelloWorld.cs)  
-  An introduction to processing events using the Event Processor, illustrating how to configure the processor and connect to the Event Hubs service.
+  An introduction to the Event Processor client, illustrating how to create the client and perform basic operations.
```

Comment:
Sample links are broken.

Reply:
That's because they haven't been merged yet. Here's the PR: #8938

Reply:
Yes; they depend on the other open [PR](https://github.com/Azure/azure-sdk-for-net/pull/8938) that adds the samples.   The links won't work until that PR closes as the samples don't exist in the repository.